crates/red_knot_python_semantic/src/module_resolver/resolver.rs

```diff
@@ -333,9 +367,8 @@ struct PthFile<'db> {
 impl<'db> PthFile<'db> {
     /// Yield paths in this `.pth` file that appear to represent editable installations,
     /// and should therefore be added as module-resolution search paths.
-    fn editable_installations(&'db self) -> impl Iterator<Item = SearchPath> + 'db {
+    fn editable_installations(&'db self) -> impl Iterator<Item = SystemPathBuf> + 'db {
```

Comment:
Hmm, I wonder if the name of this method should change now. It no longer yields fully validated editable-installation search paths; some additional validation is now needed for each path it yields. Maybe `editable_install_candidates`? Not sure...

Reply:
Feels too long to me. The terminology used by Python is to call these `items`

> A path configuration file is a file whose name has the form name.pth and exists in one of the four directories mentioned above; its contents are additional items (one per line) to be added to sys.path.

Reply:
just calling the method `items()` works for me